src/screens/Login/index.tsx

```
@@ -0,0 +1,14 @@
+import { useNavigation } from "@react-navigation/native";
```

ESLint

[error] 'useNavigation' is defined but never used.

(@typescript-eslint/no-unused-vars)